### PR TITLE
Apply and enforce formatter

### PR DESCRIPTION
**Pull request by charleskawczynski**

### CODE REVIEW

File: src/DomainDecompositions/indexes.jl [hunk 1]
```diff
@@ -60,39 +60,63 @@ Tuple of indexes including
...
-    return "_ud_"*string(i)
-  else
-    throw(BoundsError(vm[ϕ], i))
-  end
+    if i == gridmean(idx)
+        return "_gm"
+    elseif i == environment(idx)
+        return "_en"
+    elseif i in updraft(idx)
+        return "_ud_" * string(i)
```
Comment (yairchn) [inline src/DomainDecompositions/indexes.jl:93]: is this space between * and string ok?

**Review by yairchn — APPROVED** [review]
I left one comment - if that is ok than I think we can r+

File: src/DomainDecompositions/indexes.jl (hunk 1 above)
Comment (charleskawczynski) [reply src/DomainDecompositions/indexes.jl:93]: Yeah, this is fine.

**Pull request merged**